setupTests.js

```diff
@@ -0,0 +1,3 @@
+console.log('Setting up global test stubs...');
+
+URL.createObjectURL = jest.fn();
```

Comment:
why do you need this?

Reply:
Initially rendering the component prompts the error `URL.createObjectURL is not a function`.
I believe some dependency is required `createObjectURL` method. So I just mock it into global jest.

Reply:
why global instead of just for the specific test(s) ?

Reply:
I did try to move the specific test, but it won't work. 

I found some good explanations here.
https://github.com/mapbox/mapbox-gl-js/issues/9889#issuecomment-664633443